openpype/hosts/max/plugins/publish/extract_max_scene_raw.py

```diff
@@ -37,26 +28,23 @@ def process(self, instance):
         filename = "{name}.max".format(**instance.data)
 
         max_path = os.path.join(stagingdir, filename)
-        self.log.info("Writing max file '%s' to '%s'" % (filename,
-                                                         max_path))
+        self.log.info("Writing max file '%s' to '%s'" % (filename, max_path))
 
         if "representations" not in instance.data:
             instance.data["representations"] = []
 
-        # saving max scene
-        with maintained_selection():
-            # need to figure out how to select the camera
-            rt.select(get_all_children(rt.getNodeByName(container)))
-            rt.execute(f'saveNodes selection "{max_path}" quiet:true')
+        nodes = get_all_children(rt.getNodeByName(container))
+        rt.saveNodes(nodes, max_path, quiet=True)
```

Comment:
We implement` maintained_selection()` from context manager in other model extractors to restore the current selection before exporting. 
```suggestion

        with maintained_selection():
            nodes = get_all_children(rt.getNodeByName(container))
            rt.saveNodes(nodes, max_path, quiet=True)

```

Reply:
i see. that makes sense